.github/workflows/prepare-release.yml

```diff
@@ -92,7 +98,7 @@ jobs:
           distribution: "temurin"
 
       - name: Set up Gradle
-        uses: gradle/actions/setup-gradle@v3
+        uses: gradle/actions/setup-gradle@v4
```

Comment:
There are other places we use `gradle/actions/setup-gradle@v3`. Should update those too.

Reply:
Done